test/verify/check-connection

```diff
@@ -722,7 +722,7 @@ until pgrep -f cockpit-bridge.*--privileged; do sleep 1; done
             return False
         wait(check_wss_log)
 
-        wait(lambda: "received request from bad Origin" in m.execute("journalctl -b -t cockpit-ws"))
+        wait(lambda: m.execute("grep 'received request from bad Origin' /var/log/ws-notls.log"))
```

Comment:
This is ok IMHO, but needs special-casing for [distropkg](https://logs.cockpit-project.org/logs/pull-14390-20200722-204831-d902eeb3-rhel-8-3-distropkg/log.html#302)